dask/dataframe/utils.py

```diff
@@ -444,6 +466,26 @@ def _nonempty_series(s, idx=None):
             cats = None
         data = pd.Categorical(data, categories=cats,
                               ordered=s.cat.ordered)
+    elif is_integer_na_dtype(dtype):
+        data = pd.array([1, None], dtype=dtype)
+    elif is_period_dtype(dtype):
+        # pandas 0.24.0+ should infer this to be Series[Period[freq]]
```

Comment:
It's not clear from the comment, but for pandas<0.24.0, it's impossible to have a period-dtype Series. So pandas 0.24.0+ is the only case we need to worry about here.